WebRoot/skins/_base/base4/skin.properties

```diff
@@ -457,11 +457,12 @@ ZFieldBg-disabled           = background-color:transparent;
 #####################
 #   Inputs: input type=[text|password] elements only
 #####################
-InputBorder                 = border:1px solid; border-color:@darken(AppC,75)@ @darken(AppC,10)@ @darken(AppC,10)@ @darken(AppC,75)@;
+InputBorder                 = border:1px solid #CCC;
+InputHeight                 = height:32px; height:2.65rem;
```

Comment:
It may be generic in future. I am not sure of that for now. I saw similar property declarations conventions around this code. And some of those properties are directly being used in other css files. So to be in consistent here, I followed same convention.

Let me know if you have any questions. 

Thanks